index.bs

```diff
@@ -1150,10 +1150,11 @@ run the following steps:
 1. Let |attributes| be a new [=/list=].
 1. If |domain| is not null, then run these steps:
     1. If |domain| starts with U+002E (`.`), then return failure.
-    1. If |host| does not equal |domain| and
-        |host| does not end with U+002E (`.`) followed by |domain|,
+    1. Let |parsedDomain| be the result of [=host parser|host parsing=] |domain|.
```

Comment:
We should consider the case when host parsing returns a failure, since that is a possibility [according to the spec](https://url.spec.whatwg.org/#host-parsing).

Reply:
I added another line in the latest commit. PTAL!